src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -110,6 +110,26 @@
  *   &lt;property name=&quot;reliefPattern&quot; value=&quot;continue in next case&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Violation</p>
+ * <pre>
+ *     switch (i) {
+ *      case 0:
+ *      case 1:
+ *          i++;
+ *      case 2: //Violation
+ *    }
+ * </pre>
+ * <p>acceptable code</p>
+ * <pre>
+ *     switch (i) {
+ *      case 0:
+ *          i++;
+ *          break;
+ *      case 1:
+ *          i++;
+ *          //fall through
```

Comment:
like @muhlba91 mentioned, this doesn't match the configuration, which uses a custom `reliefPattern` - please update the config/example code so they align